### ISSUE TITLE
SBIE2203 Failed to communicate with Sandboxie Service:  *MANPROXY

### ISSUE THREAD

**Issue by ganego** [sandboxie-plus/Sandboxie#643]
Hello,

first of all, thank you for continuing to develop Sandboxie.  
I use Sandboxie Classic x64 on Windows 7. I recently updated to 5.48.0 and since then when I use a sandboxed PotPlayer to play a video I get the following errors. This was not the case in the past, although I do not know anymore what version I used before (I think it was 5.46.1 since I found it in my download log).  
Programs are not allowed to access the internet, admin rights are dropped, network files are blocked.

> SBIE2203 Failed to communicate with Sandboxie Service:  *
> SBIE2203 Failed to communicate with Sandboxie Service:  *MA
> SBIE2203 Failed to communicate with Sandboxie Service:  *MANP
> SBIE2203 Failed to communicate with Sandboxie Service:  *MANP
> SBIE2203 Failed to communicate with Sandboxie Service:  *MANPR
> SBIE2203 Failed to communicate with Sandboxie Service:  *MANPRO
> SBIE2203 Failed to communicate with Sandboxie Service:  
> SBIE2203 Failed to communicate with Sandboxie Service:  *MANPROX
> SBIE2203 Failed to communicate with Sandboxie Service:  *
> SBIE2203 Failed to communicate with Sandboxie Service:  *
> SBIE2203 Failed to communicate with Sandboxie Service:  *MANPROXY
> SBIE2203 Failed to communicate with Sandboxie Service:  *M
> SBIE2203 Failed to communicate with Sandboxie Service:  *MA
> SBIE2203 Failed to communicate with Sandboxie Service:  *MA
> SBIE2203 Failed to communicate with Sandboxie Service:  *MAN
> SBIE2203 Failed to communicate with Sandboxie Service:  *MAN
> SBIE2203 Failed to communicate with Sandboxie Service:  *MANPRO
> SBIE2203 Failed to communicate with Sandboxie Service:  
> SBIE2203 Failed to communicate with Sandboxie Service:  *MANPROX

Any idea why this suddently shows up? Does this "MANPROXY" belong to Sandboxie or PotPlayer?  
Was there something not catched before or is there just now an error connected to it?  
PotPlayer seems to work fine regardless, so I could just hide the messages, but still I'd like to know why they suddenly appear.

Thank you

**Comment by DavidXanatos** [comment]
could you please test older versions and pinpoint which version introduced the error

**Comment by ganego** [comment]
tl;dr: started with 5.45.0.

I downgraded/overwrite my Sandboxie with 5.43.5 then installed every consecutive version and they worked fine, but v5.45.0.for.windows.7.exe did not start the driver and had errors. I then installed 5.45.1 and it had the MANPROXY errors. I then installed 5.45.0 over it (which worked) and also had MANPROXY errors although now they had encoding errors I guess, see log:

> SBIE2203 Failed to communicate with Sandboxie Service:  *MANPROXY⒔
> SBIE2203 Failed to communicate with Sandboxie Service:  *M⒔
> SBIE2203 Failed to communicate with Sandboxie Service:  *MA⒔
> SBIE2203 Failed to communicate with Sandboxie Service:  *MAN⒔
> SBIE2203 Failed to communicate with Sandboxie Service:  *MANP⒔

Errors in some later version with also encoding error.
> SBIE2203 Failed to communicate with Sandboxie Service:  *MAN┠
> SBIE2203 Failed to communicate with Sandboxie Service:  *MANP┠
> SBIE2203 Failed to communicate with Sandboxie Service:  *MANPR┠
> SBIE2203 Failed to communicate with Sandboxie Service:  *MANPRO┠
> SBIE2203 Failed to communicate with Sandboxie Service:  ┠

**Comment by ganego** [comment]
It has something to do with File Migration.  
I now installed 5.49.0 and with my default file migration setting (don't migrate larger than 5 KB) it gives MANPROXY error. When I set it to 1000 KB, there is no error.

**Comment by DavidXanatos** [comment]
@ganego could you please test the last build and report back

**Comment by DavidXanatos** [comment]
This is very strange, i can not reproduce this log corruption
but i think you can get rid of it by 
removing ```PromptForInternetAccess=y```
from your config

the MANPROXY queue is available only when using the plus ui, the classical ui should filter that error and not show it, but as the MANPROXY string for whatever reason gets broken it fails to ignore it as it does not match.

**Comment by ganego** [comment]
> removing PromptForInternetAccess=y  from your config

Does not exist in sandboxie.ini

```
[Media_No_Internet]

Enabled=y
ConfigLevel=8
AutoRecover=y
BlockNetworkFiles=y
Template=SkipHook
Template=FileCopy
Template=qWave
Template=WindowsFontCache
Template=BlockPorts
Template=LingerPrograms
Template=Chrome_Phishing_DirectAccess
Template=AutoRecoverIgnore
RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
RecoverFolder=%Personal%
RecoverFolder=%Favorites%
RecoverFolder=%Desktop%
BorderColor=#00ffff,off,6
BoxNameTitle=n
ForceProcess=<PotPlayer>
ForceProcess=vlc.exe
NotifyInternetAccessDenied=n
ProcessGroup=<PotPlayer>,desktophook.exe,deskto~1.exe,desktophook64.exe,deskto~2.exe,killpot64.exe,killpo~1.exe,potplayermini64.exe,potpla~1.exe
DropAdminRights=y
ClosedFilePath=InternetAccessDevices
CopyLimitKb=600
CopyLimitSilent=y
OpenFilePath=%AppData%\PotPlayerMini64\
BlockNetParam=y
FakeAdminRights=n
ClosePrintSpooler=y
OpenPrintSpooler=n
AllowSpoolerPrintToFile=n
OpenSmartCard=n
NeverDelete=n
AutoDelete=n
AllowRawDiskRead=n
NotifyDirectDiskAccess=n
```

I use the classic UI because it uses less RAM and because the new UI spams me with these messages to copy a file for whatever reason. Oh, and also: MANPROXY...

![image](https://user-images.githubusercontent.com/4750881/113208850-95acf800-9272-11eb-8621-92ccde5efadd.png)

**Comment by DavidXanatos** [comment]
I can not reproduce the ma proxy issue, for some reason the communication with the plus UI fails than this error is displayed.
In the next build you will have the option to disable these file copy prompts, forcing the behavioure of the classical UI that is denying the copy operation right away.
setting will be ```PromptForFileMigration=n```
